modules/samples/s0/scripts/sample.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var log = new Log();
+
+try {
+    var dashboard = require("/modules/dashboards.js");
+    var config = require("/configs/sample.json");
+} catch (Exception) {
```

Comment:
Catch specific exception, and log the exception that is thrown. Currently there is no information extracted from the original exception that its thrown.